opennms-config/src/main/resources/META-INF/opennms/applicationContext-commonConfigs.xml

```diff
@@ -166,4 +166,10 @@
 
   <onmsgi:service interface="org.opennms.netmgt.config.api.ResourceTypesDao" ref="resourceTypesDao" />
 
+  <onmsgi:list id="contextServiceMonitor"
+	 interface="org.opennms.netmgt.poller.ServiceMonitor">
+	  <onmsgi:listener ref="pollerConfig"
+		bind-method="onBind" unbind-method="unBind"/>
```

Comment:
Can you change these methods to onServiceMonitorBind() and onServiceMonitorUnbind()? Thanks.